models/models.go

```diff
@@ -785,22 +795,22 @@ func (m *RouteManager) deployCertificate(route Route, cert acme.CertificateResou
 		"instance-id": route.InstanceId,
 	})
 
+	lsession.Info("get-cert-expiry")
 	expires, err := acme.GetPEMCertExpiration(cert.Certificate)
 	if err != nil {
-		lsession.Error("get-cert-expiry", err)
+		lsession.Error("get-cert-expiry-err", err)
```

Comment:
Can we include the domain here?

Reply:
Resolved by including the domains in all this function via the `logger.Session` call.